webapp/src/components/channel_header_button/channel_header_button.tsx

```diff
@@ -4,12 +4,12 @@
 import React from 'react';
 
 type Props = {
-    shouldHighlight: boolean
+    isVisible: boolean,
```

Comment:
I think this prop should still be called `shouldHighlight`, because that's all we want to convey to the channel header button.